premake5.lua

```diff
@@ -22,6 +22,7 @@ newoption { trigger = "no-build-lua", category = "YGOPro - lua", description = "
 newoption { trigger = "lua-include-dir", category = "YGOPro - lua", description = "", value = "PATH" }
 newoption { trigger = "lua-lib-dir", category = "YGOPro - lua", description = "", value = "PATH" }
 newoption { trigger = "lua-lib-name", category = "YGOPro - lua", description = "", value = "NAME", default = LUA_LIB_NAME }
+newoption { trigger = "lua-deb", category = "YGOPro - lua", description = "Use Debian lua package" }
```

Comment:
lua-use-lua-cpp-from-package-manager

Reply:
Not very interested in maintaining support on all common Linux distros.

Plus, some distros like Fedora does not have Lua in C++